add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The script uses `mysqli` for database connection. However, it lacks an abstraction layer for database operations.

Recommendations: Implement a Repository pattern to abstract database operations, making the codebase more maintainable and testable.

Example:
```php
class BookRepository {
    private $connection;

    public function __construct($connection) {
        $this->connection = $connection;
    }

    public function addBook($title, $author, $year) {
        $stmt = $this->connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
        $stmt->bind_param("ssi", $title, $author, $year);
        return $stmt->execute();
    }
}
```